Riot/Modules/MatrixKit/Utils/EventFormatter/MarkdownToHTMLRenderer.swift

```diff
@@ -44,7 +44,7 @@ extension MarkdownToHTMLRenderer: MarkdownToHTMLRendererProtocol {
             ast.repairLinks()
             return try DownHTMLRenderer.astToHTML(ast, options: options)
         } catch {
-            MXLog.error("[MarkdownToHTMLRenderer] renderToHTML failed with string: \(markdown)")
+            MXLog.error("[MarkdownToHTMLRenderer] renderToHTML failed")
```

Comment:
Not logging the `markdown` in case it contains sensitive decrypted data. If we get a lot of events without context, we can add `mxlog.debug` for rageshakes